Exercises/operators-and-type-conversions/operators-and-type-conversions.js

```diff
@@ -0,0 +1,21 @@
+// Task #1
+a = 10;
+console.log(parseInt(a + '0'));
+
+// Task #2
+const stringValue = `Amazon annual revenue of US$232.887 billion`;
+let r = new RegExp(/\d+.\d+/);
+let annualRevenue = +stringValue.match(r);
+if (typeof annualRevenue === "number") {
+    console.log(`Annual revenue is: ${annualRevenue *= 1e9}`);
+    const monthlyIncome = annualRevenue / 12;
+    console.log(`Monthly income is: ${Math.round(monthlyIncome)}`);
+}
+else {
```

Comment:
else should be placed by next pattern:
```
if () {
} else {}
```

Reply:
Sorry, I didn't understand. Did you mean to replace else with another if condition?